packages/core/src/services/chains/ChainValidator/index.ts

```diff
@@ -160,7 +175,7 @@ export const validateChain = async (
 
   const rule = applyCustomRules({
     providerType: provider.provider,
-    messages: conversation.messages,
+    messages: conversation.messages as Message[],
```

Comment:
Why this? Is not the same signature?

Reply:
Different types from different packages with different enums (although same values). Typescript does not like that.